### DISCUSSION TITLE
How to enable the woodpecker manager?

### DISCUSSION THREAD

**Discussion by OCram85** [renovatebot/renovate#19302] [Request Help] [answered]
Does anyone know how to enable the woodpecker manager?

It seems like I missed anything, because the default woodpecker pipeline files doesn't get recognized in `.woodpecker/.*.yml`.

Here is an example of my commonly used basic config:

- `renovate.json`
  - https://gitea.ocram85.com/CodeServer/arkanum/src/branch/master/renovate.json
  - https://gitea.ocram85.com/OCram85/Blog/src/branch/master/renovate.json
- woodpecker pipeline to run containerized renovate 
  - https://gitea.ocram85.com/CodeServer/arkanum/src/branch/master/.woodpecker/.renovate.yml
  - https://gitea.ocram85.com/OCram85/Blog/src/branch/master/.woodpecker/.renovate.yml

**Comment by rarkins** [comment, marked as answer]
Why are you overriding the default `fileMatch` for woodpecker in the first place? It looks like you are not escaping your regex correctly

**Reply by OCram85** [reply]
I tried to debug  the file detection, because the defaults didn't find anything.

**Reply by rarkins** [reply]
What's an example of a file name you expect to match?

**Reply by OCram85** [reply]
I use the woodpecker default file pattern lik this:

```
.woodpecker/pipeline1.yml
.woodpecker/pipeline2.yml
```

Just found in the woodpecker docks that you can also use a `.` as prefix in the pipeline file names like `.woodpecker/.master.yml`

But both patterns doesn't seem to match - I tested both

Here is the renovate debug log from recent build without custom fileMatch:

- https://ci.ocram85.com/OCram85/Blog/build/108/12

![image](https://user-images.githubusercontent.com/13993700/206401219-30547875-0cda-4f85-a06a-46febf1fdb7a.png)

**Reply by rarkins** [reply]
I'm guessing the pattern was based off this naming convention here: https://woodpecker-ci.org/docs/usage/intro#configuration

**Reply by rarkins** [reply]
If you want to override it, try `^.woodpecker\/.*\.yml$`

**Reply by OCram85** [reply]
Found an issue in my manager config: I defined the `fileMatch` as `string` and not as *string array*.

This regex pattern seems to work now:

```
  "woodpecker": {
    "enabled": true,
    "fileMatch": ["^.woodpecker\/.*.yml$"]
  }
```
